test/jdk/TEST.groups

```diff
@@ -635,13 +638,23 @@ jdk_core_manual_no_input_security = \
     sun/security/smartcardio/TestMultiplePresent.java \
     sun/security/smartcardio/TestPresent.java \
     sun/security/smartcardio/TestTransmit.java \
-    sun/security/tools/jarsigner/compatibility/Compatibility.java \
-    java/security/Policy/Root/Root.java
+    sun/security/tools/jarsigner/compatibility/Compatibility.java
+
+jdk_core_manual_human = \
+    :jdk_core_manual_requires_human_input \
```

Comment:
Move tests from jdk_core_manual_requires_human_input group to here. And remove jdk_core_manual_requires_human_input this group.